src/endpoints/Billing/types.ts

```diff
@@ -0,0 +1,18 @@
+export interface SignupRequest {
+    email: string;
+    company_name: string;
+    first_name: string;
+    last_name: string;
+    plan: 'core';
+    currency: 'eur';
+    quantity: number;
+    billing_cycle: 'month';
```

Comment:
`plan`, `currency` and `billing_cycle` shouldn't be hardcoded. We were asked to provide ways to allow to sign up for any plan.

Reply:
@aspirisen this one still stands. [Here is the code](https://github.com/prezly/prezly/blob/86312d46d76157f2b5d6a9714a13423522af1f83/lib/Prezly/Support/Plans/Plan.php#L28) that describes all supported values for `plan`, `currency` and `billing_cycle`